enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline_one' not handled in switch

### Zig Version

0.10.0-dev.4247+3234e8de3

### Zig Language Server Version

ac6353add7b1dc5fdee25333b02d8c52e589502a

### Steps to Reproduce

zig build -Drelease-safe

### Expected Behavior

Should compile

### Actual Behavior

./src/analysis.zig:2519:5: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline_one' not handled in switch
    switch (node_tag) {
    ^
./src/analysis.zig:2519:5: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline' not handled in switch
    switch (node_tag) {
    ^
./src/semantic_tokens.zig:299:5: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline_one' not handled in switch
    switch (tag) {
    ^
./src/semantic_tokens.zig:299:5: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline' not handled in switch
    switch (tag) {
    ^
./src/ast.zig:265:18: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline_one' not handled in switch
    while (true) switch (tags[n]) {
                 ^
./src/ast.zig:265:18: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline' not handled in switch
    while (true) switch (tags[n]) {
                 ^
./src/inlay_hints.zig:277:5: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline_one' not handled in switch
    switch (tag) {
    ^
./src/inlay_hints.zig:277:5: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline' not handled in switch
    switch (tag) {
    ^
./src/references.zig:95:5: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline_one' not handled in switch
    switch (node_tags[node]) {
    ^
./src/references.zig:95:5: error: enumeration value 'std.zig.Ast.Node.Tag.switch_case_inline' not handled in switch
    switch (node_tags[node]) {

--- a/src/analysis.zig
+++ b/src/analysis.zig
@@ -2828,7 +2828,9 @@ fn makeScopeInternal(allocator: std.mem.Allocator, context: ScopeContext, node_i
             }
         },
         .switch_case,
+        .switch_case_inline,
         .switch_case_one,
+        .switch_case_inline_one,
         .switch_range,
         => {
             return;

--- a/src/ast.zig
+++ b/src/ast.zig
@@ -437,6 +437,8 @@ pub fn lastToken(tree: Ast, node: Ast.Node.Index) Ast.TokenIndex {
         .array_type,
         .switch_case_one,
         .switch_case,
+        .switch_case_inline_one,
+        .switch_case_inline,
         .switch_range,
         => n = datas[n].rhs,
 

--- a/src/inlay_hints.zig
+++ b/src/inlay_hints.zig
@@ -548,8 +548,10 @@ fn writeNodeInlayHint(builder: *Builder, arena: *std.heap.ArenaAllocator, store:
 
         .switch_case_one,
         .switch_case,
+        .switch_case_inline_one,
+        .switch_case_inline,
         => {
-            const switch_case = if (tag == .switch_case) tree.switchCase(node) else tree.switchCaseOne(node);
+            const switch_case = if (tag == .switch_case or tag == .switch_case_inline) tree.switchCase(node) else tree.switchCaseOne(node);
 
             try callWriteNodeInlayHint(allocator, .{ builder, arena, store, switch_case.ast.target_expr, range });
         },

--- a/src/references.zig
+++ b/src/references.zig
@@ -176,13 +176,17 @@ fn symbolReferencesInternal(
                 try symbolReferencesInternal(builder, case, handle);
             }
         },
-        .switch_case_one => {
+        .switch_case_one,
+        .switch_case_inline_one,
+        => {
             const case_one = tree.switchCaseOne(node);
             try symbolReferencesInternal(builder, case_one.ast.target_expr, handle);
             for (case_one.ast.values) |val|
                 try symbolReferencesInternal(builder, val, handle);
         },
-        .switch_case => {
+        .switch_case,
+        .switch_case_inline,
+        => {
             const case = tree.switchCase(node);
             try symbolReferencesInternal(builder, case.ast.target_expr, handle);
             for (case.ast.values) |val|

--- a/src/semantic_tokens.zig
+++ b/src/semantic_tokens.zig
@@ -522,6 +522,7 @@ fn writeNodeTokens(builder: *Builder, maybe_node: ?Ast.Node.Index) WriteTokensEr
         .switch_case_inline,
         => {
             const switch_case = if (tag == .switch_case or tag == .switch_case_inline) tree.switchCase(node) else tree.switchCaseOne(node);
+            try writeToken(builder, switch_case.inline_token, .keyword);
             for (switch_case.ast.values) |item_node| try callWriteNodeTokens(allocator, .{ builder, item_node });
             // check it it's 'else'
             if (switch_case.ast.values.len == 0) try writeToken(builder, switch_case.ast.arrow_token - 1, .keyword);

--- a/src/semantic_tokens.zig
+++ b/src/semantic_tokens.zig
@@ -518,8 +518,10 @@ fn writeNodeTokens(builder: *Builder, maybe_node: ?Ast.Node.Index) WriteTokensEr
         },
         .switch_case_one,
         .switch_case,
+        .switch_case_inline_one,
+        .switch_case_inline,
         => {
-            const switch_case = if (tag == .switch_case) tree.switchCase(node) else tree.switchCaseOne(node);
+            const switch_case = if (tag == .switch_case or tag == .switch_case_inline) tree.switchCase(node) else tree.switchCaseOne(node);
             for (switch_case.ast.values) |item_node| try callWriteNodeTokens(allocator, .{ builder, item_node });
             // check it it's 'else'
             if (switch_case.ast.values.len == 0) try writeToken(builder, switch_case.ast.arrow_token - 1, .keyword);

